offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -161,7 +164,11 @@ class QuotationDetails {
         addSubTotalPrices(productGroup, subTotal)
     }
 
-    private String createDiscountDescription(double quantity, String unit, double discountPercentage) {
+    private String createDataStorageDiscountDescription() {
+        return "Data storage (item no ${itemNumber}) is free of charge due to internal funding."
```

Comment:
```suggestion
        return "Data storage on item no ${itemNumber} is free of charge due to internal funding."
```
Maybe try to formulate/organize the sentence structure similar to the other discount description text :)

Reply:
uh nice